### PR TITLE
Indoor main page cu 862jgqf75

### PR DESCRIPTION
**Pull request by andrewmarushko**

### CODE REVIEW

**Comment by andrewmarushko** [comment]
Task linked: [CU-862jgqf75 Indoor Main Page](https://app.clickup.com/t/862jgqf75)

File: app/indoor/page.tsx [hunk 1]
```diff
@@ -1,27 +1,57 @@
...
-            <h3 className='font-extrabold text-2xl'>{data.country}</h3>
-            <div>
-              {data.data.map(({id, name}) => (
-                <p key={id}>
-                  {name}
-                </p>
-              ))}
+      <div className="grid gap-10">
+        {data &&
+          data.map((countryData: any) => (
```
Comment (OleksandrMykhailovskyi) [inline app/indoor/page.tsx:35]: you can use IndoorDataListInterface interface here

File: app/indoor/page.tsx [hunk 2]
```diff
@@ -1,27 +1,57 @@
...
+import { Button } from '@/components/ui/button';
+
+import Page from '@/components/ui/page';
+
+export interface Indoor {
+  id: number;
+  attributes: Attributes;
+}
+
+export interface Attributes {
```
Comment (OleksandrMykhailovskyi) [inline app/indoor/page.tsx:12]: it seems IndoorDataItemInterface is more detailed one

File: app/indoor/page.tsx [hunk 3]
```diff
@@ -1,27 +1,57 @@
...
-                <p key={id}>
-                  {name}
-                </p>
-              ))}
+      <div className="grid gap-10">
+        {data &&
+          data.map((countryData: any) => (
+            <div key={countryData.country} className="grid grid-cols-4 gap-12">
+              {countryData &&
+                countryData.data.map((indoor: any) => (
```
Comment (OleksandrMykhailovskyi) [inline app/indoor/page.tsx:38]: here is the interface that can be used instead of any IndoorDataItemInterface

Comment (OleksandrMykhailovskyi) [reply app/indoor/page.tsx]: fixed

File: app/indoor/page.tsx (hunk 2 above)
Comment (OleksandrMykhailovskyi) [reply app/indoor/page.tsx]: removed unused Attributes and Indoors interfaces

File: app/indoor/page.tsx (hunk 1 above)
Comment (OleksandrMykhailovskyi) [reply app/indoor/page.tsx]: removed any and changed to the existing interface

**Review by OleksandrMykhailovskyi — APPROVED** [review]

**Review by andrewmarushko — COMMENTED** [review]
Approve please  @OleksandrMykhailovskyi @iznihyaAnton

**Pull request merged**